### PR TITLE
fix: resolves csv uploader parsing issue by using native methods

### LINKED ISSUE #920: Idea: Stop parsing CSV in the UI
We have mechanisms to upload csv data into the UI. We have custom parsers for that code.

This issue it to replace our javascript based parsing with calls to the query api using the `csv.from` and `to` syntax for flux: https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/csv/from/

the experience could be similar to dropping an image into a github issue. User could drop an LP file into our UI, and we upload the file and give it a unique url. then we run `csv.from(url: "that new url")`

We could do that on any code editor in the UI.

### PR DESCRIPTION
**Pull request by Asalem1**
Closes #920
Closes https://github.com/influxdata/influxdb/issues/21636

This PR makes a change to the implementation for the CSV Uploader to use the native csv.from rather than parsing the CSV and writing is a line protocol in 6 concurrent streams. After doing some testing it seems like the speed with which the 6 concurrent streams were being written is only marginally faster than writing using the csv.from implementation. In doing so, this resolves issues that have come up due to parsing. It also allows users to upload csv that isn't properly formatted according to line protocol.

This PR also fixes an issue with the abort controller reference where users that cancelled a query would have to reload the page in order to get a new abort controller for the query. The issue manifested itself after a query was aborted and a user tried to upload a different file (the original abort controller being in a cancelled state wouldn't allow for subsequent uploads). 

Finally, this PR adds an actionable button when users have finished uploading their CSV so that they can just click to navigate directly to their bucket. Ideally this would link to a notebook we'd create for them, but I think that's a ways a way at this point.

csv-uploader

### CODE REVIEW

**Comment by Asalem1** [comment]
@russorat can you verify that this may close #1544 or did you have something else in mind?

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1996)

**Comment by russorat** [comment]
i ❤️ this!

File: src/writeData/components/fileUploads/CsvMethod.tsx [hunk 1]
```diff
@@ -46,11 +41,19 @@ const CsvMethod: FC = () => {
...
   }
 
+  let body = <CsvUploaderBody bucket={bucket.name} />
+
+  if (uploadState !== RemoteDataState.NotStarted) {
+    body = <StatusIndicator />
+  }
+
+  const handleSeeUploadedData = () => {
+    history.push(`/orgs/${orgId}/data-explorer?bucket=${bucket.name}`)
```
Comment (drdelambre) [inline src/writeData/components/fileUploads/CsvMethod.tsx:51]: IM COMING FOR YOU LINK

Comment (Asalem1) [reply src/writeData/components/fileUploads/CsvMethod.tsx:51]: totally

**Review by drdelambre — APPROVED** [review]
RIP

File: cypress/e2e/shared/buckets.test.ts [hunk 2]
```diff
@@ -108,100 +108,6 @@ describe('Buckets', () => {
...
-    })
-
-    // this is currently not producing success, its actually failing, im going to write a separate issue for this
-    it('closes the overlay upon a successful delete with predicate submission', () => {
-      cy.getByTestID('delete-checkbox').check({force: true})
-      cy.getByTestID('confirm-delete-btn').click()
-      cy.getByTestID('overlay--container').should('not.exist')
-      cy.getByTestID('notification-success').should('have.length', 1)
-    })
-  })
```
Comment (Asalem1) [inline cypress/e2e/shared/buckets.test.ts:204]: deleted those since it ain't gonna happen

**Comment by kyleawayan** [comment]
Amazing! Thank you!

**Pull request merged**